ruby_connectors/spec/duckdb_connector/client_spec.rb

```
@@ -5,31 +5,31 @@
         }
       }
     }
   end
 
-  subject(:client) { described_class.new(config) }
-
-  describe '#initialize' do
-    it 'creates a new connection' do
+  describe "#initialize" do
+    it "creates a new connection" do
       expect(RubyConnectors::DuckdbConnector::Connection).to receive(:new).with(config)
```

rubocop

[convention] Prefer `have_received` for setting message expectations. Setup `RubyConnectors::DuckdbConnector::Connection` as a spy using `allow` or `instance_spy`.

(RSpec/MessageSpies)